ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -414,6 +414,8 @@ objects:
               vmType: server
               instanceTypeID: 
           replicas: 2
+        networking:
+          machineCIDR: 192.168.0.0/16
```

Comment:
This field is deprecated; I think you should be using `networking.machineNetwork`

Reply:
done, let's see if this pass  the installation.